controllers/flp/flp_pipeline_builder.go

```diff
@@ -72,6 +72,9 @@ func (b *PipelineBuilder) AddProcessorStages() error {
 			Type:   api.AddKubernetesRuleType,
 		}, {
 			Type: api.ReinterpretDirectionRuleType,
+		}, {
+			Type:       "add_kubernetes_infra",
```

Comment:
TODO once FLP PR is merged: update FLP dependency and change this string using FLP API